packages/math-inline/src/main.jsx

```diff
@@ -299,17 +307,70 @@ export class Main extends React.Component {
             });
         }
     };
+
+    focusFirstKeypadElement = () => {
+        setTimeout(() => {
+          const keypadElement = document.querySelector('[data-keypad]');
+
+          if (keypadElement) {
+            const firstButton = keypadElement.querySelector("button, [role='button']");
+
+            if (firstButton) {
+              firstButton.focus();
+            }
+          }
+        }, 0);
+    };
+
+    handleKeyDown = (event, id) => {
+        const isAnswerInputFocused = document.activeElement.getAttribute('aria-label') === 'Enter answer.';
+        const isClickOrTouchEvent = event.type === 'click' || event.type === 'touchstart';
+    
+        if (isAnswerInputFocused && (event.key === 'ArrowDown' || isClickOrTouchEvent)) {
+          this.setState({ activeAnswerBlock: id }, () => {
+            if (event.key === 'ArrowDown') {
+              this.focusFirstKeypadElement();
+            }
+          });
+        }
+    
+        if (!isAnswerInputFocused && isClickOrTouchEvent) {
+          this.setState({ activeAnswerBlock: '' });
+        }
+    
+        if (event.key === 'Escape') {
+          this.setState({ activeAnswerBlock: '' });
+        }
```

Comment:
This could be just one if statement since it has the same body
```
if ((!isAnswerInputFocused && isClickOrTouchEvent) || event.key === 'Escape') {
          this.setState({ activeAnswerBlock: '' });
        }
```